README.md

```diff
@@ -10,6 +10,7 @@ CONerd Quotebot is this idea that any message spoken in a Discord server could b
 - [x] Build CI/CD pipeline
 - [x] Determine the database technology (CosmosDb)
 - [x] Add database service implementation
+  - [ ] Determine table schema (Date/Time, messageID (primary key), username, message, etc.)
```

Comment:
CosmosDb is schema less. Perhaps a better to do item might be to determine the partition key for the quote collection.